src/core/hoc/StylesInjection.tsx

```diff
@@ -3,15 +3,17 @@ import { StyleSheetManager } from 'styled-components';
 import createCache from '@emotion/cache';
 import { CacheProvider } from '@emotion/react';
 
-interface Props extends React.HTMLProps<HTMLDivElement> {
-    id: string;
+interface Props extends Pick<React.HTMLProps<HTMLDivElement>, 'children'> {
     injectionPoint?: HTMLElement | undefined;
 }
 
 export const StylesInjection = (props: Props) => {
 
+  // @emotion/cache only accept alpha characters
+  const uniqueAlphaCharacterId = Math.random().toString(36).replace(/[^a-z]+/g, '').substring(0, 10);
+
   const emotionCache = createCache({
-    key: props.id,
+    key: uniqueAlphaCharacterId,
```

Comment:
IMHO, I would prefer to generate a unique ID if it is not provided in the props. That way there is a way to provide it if needed.

Reply:
Resolved, thanks.